rocrand/PKGBUILD

```diff
@@ -34,4 +34,5 @@ package() {
 /opt/rocm/rocrand/lib
 EOF
   install -Dm644 "$srcdir/rocRAND-rocm-$pkgver/LICENSE.txt" "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+  ln -s "/opt/rocm/hiprand/include/hiprand.h" "$pkgdir/opt/rocm/include/hiprand/hiprand.h"
```

Comment:
Instead of just symlinking that one header file, why not link the hiprand directory? I think you should also create the /opt/rocm/include directory. Please see https://github.com/rocm-arch/rocm-arch/commit/b120d233c87dfd99362fa450e621367e5dbb4139 as a reference commit for this PR.